src/main/java/com/game/Battleship/BattleshipApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.game.Battleship;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class BattleshipApplication {
+
+	public static void main(String[] args) {
+		SpringApplication.run(BattleshipApplication.class, args);
+	}
+
```

Comment:
You have a blank line here but not in src/main/java/com/game/Battleship/controllers/HomeController.java after the `welcome()` method. Please choose whether or not to leave a blank line and be consistent with that.

Reply:
Done!